tests/model_serving/model_server/maas_billing/utils.py

```diff
@@ -0,0 +1,103 @@
+from __future__ import annotations
+
+from typing import Dict, Optional
+
+import base64
+import requests
+from ocp_resources.gateway_gateway_networking_k8s_io import Gateway
+from ocp_resources.ingress_config_openshift_io import Ingress as IngressConfig
+from pyhelper_utils.shell import run_command
+from requests import Response
+
+
+def host_from_ingress_domain(client) -> str:
+    """
+    Return 'maas.<ingress-domain>' using the wrapper (no 'oc get' calls).
+    """
+    ing = IngressConfig(name="cluster", client=client)
+    assert ing.exists, "ingresses.config.openshift.io/cluster not found"
```

Comment:
instead of this line, you can pass ensure_exists=True in the call `IngressConfig(name="cluster", client=client)`